shard.yml

```diff
@@ -1,13 +1,14 @@
 name: secrets-env
-version: 1.0.0
+version: 1.1.0
 
 development_dependencies:
   ameba:
     github: crystal-ameba/ameba
 
 authors:
   - Kim Burgess <kim@place.technology>
+  - Caspian Baska <caspian@place.technology>
 
-crystal: 0.34.0
+crystal: 0.35
```

Comment:
This shouldn't need to change. The version here specifies compatible compiler versions - everything here looks to still work with 0.34.0 and up.